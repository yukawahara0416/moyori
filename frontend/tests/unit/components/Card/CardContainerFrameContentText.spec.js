import { shallowMount } from '@vue/test-utils'
import Component from '@/components/Card/CardContainerFrameContentText.vue'

let wrapper
let propsData

beforeEach(() => {
  propsData = {
    spot: { marker: { name: 'test' }, data: { id: 1 } }
  }

  wrapper = shallowMount(Component, {
    propsData
  })
})

afterEach(() => {
  wrapper.destroy()
})

describe('props', () => {
  it('spot', () => {
    expect(wrapper.props().spot).toStrictEqual(propsData.spot)
    expect(wrapper.props().spot instanceof Object).toBe(true)
  })
})

describe('template', () => {
  it('snapshot', () => {
    expect(wrapper.vm.$el).toMatchSnapshot()
  })
})
