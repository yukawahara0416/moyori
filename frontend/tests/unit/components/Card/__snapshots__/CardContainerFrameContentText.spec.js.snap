// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`template snapshot 1`] = `
<v-card-text-stub
  class="text--primary"
>
  <p>
    test
  </p>
   
  <p>
    
  </p>
   
  <p>
    
  </p>
   
  <p>
    <a
      href="https://www.google.com/maps/dir/?api=1&destination=undefined,undefined"
      target="_brank"
    >
      
      Googleマップでナビ
    
    </a>
  </p>
</v-card-text-stub>
`;
